plenum/test/checkpoints/test_stashed_messages_processed_on_backup_replica_ordering_resumption.py

```diff
@@ -92,34 +104,36 @@ def test_stashed_messages_processed_on_backup_replica_ordering_resumption(
 
     # Ensure that the replica has not ordered any batches
     # after the very first one
-    assert slow_replica.last_ordered_3pc == (view_no, 2)
+    assert slow_replica.last_ordered_3pc == (view_no, batches_count)
 
     # Ensure that the watermarks have not been shifted since the view start
-    assert slow_replica.h == 0
-    assert slow_replica.H == LOG_SIZE
+    assert slow_replica.h == low_watermark
+    assert slow_replica.H == (sys.maxsize if first_run else low_watermark + LOG_SIZE)
 
     # Ensure that there are some quorumed stashed checkpoints
     check_num_quorumed_received_checkpoints(slow_replica, 1)
 
     # Ensure that now there are 3PC-messages stashed
     # as laying outside of the watermarks
-    assert slow_replica.stasher.stash_size(STASH_WATERMARKS) == incoming_3pc_msgs_count(len(txnPoolNodeSet))
+    if not first_run:
+        assert slow_replica.stasher.stash_size(STASH_WATERMARKS) == incoming_3pc_msgs_count(len(txnPoolNodeSet))
```

Comment:
Why don't we expect it on first run now?

Reply:
on start we set up high watermark for backup as sys.maxsize